app/models/framework/definition.rb

```diff
@@ -59,7 +59,7 @@ def for_entry_type(entry_type)
     end
 
     # Require all the framework definitions up-front
-    Dir['app/models/framework/definition/*'].each do |definition|
+    Dir['app/models/framework/definition/*.rb'].each do |definition|
```

Comment:
I was literally looking at this change in the draft branch and thinking, "this would make a good separate commit, and also `"*.rb "`"! 🙏